lib/hooks/queries/usePortfolioPositions.ts

```diff
@@ -165,9 +168,10 @@ export const usePortfolioPositions = (
   const now = useChainTime();
 
   const { data: ztgPrice } = useZtgPrice();
-  const block24HoursAgo = Math.floor(now?.block - 7200);
+  const block24HoursAgo = Math.floor(now?.block ?? 0 - 7200);
```

Comment:
If `now.block` is `undefined` or `null` this would return `NaN`, now it will return a negative number. I noticed there are quite a few `isNaN` checks, do any of these need to be modified?

Reply:
I will keep this resolving to NaN if current block isnt fetched. The queries that need `block24HoursAgo` are only enabled when the blocknumber is true, so in case of NaN it will wait.